tools/bgpreader.c

```diff
@@ -211,8 +225,10 @@ static void usage()
       fprintf(stderr, " -%c, --%-23s%-15s  %s\n", OPTIONS[k].option.val,
               OPTIONS[k].option.name, OPTIONS[k].usage, expl_buf);
     } else {
+#ifdef WITH_RPKI
```

Comment:
@salsh why add this `#ifdef`?

Reply:
yeah, i'm confused by this too. and what are the 50x numbers in the argument config structs?

Reply:
The long args should be visible in usage (help section) only if RPKI support is active. Assumption: all long args belong to RPKI.